chain/x/zoracle/client/cli/tx.go

```diff
@@ -285,3 +288,135 @@ $ %s tx zoracle edit-data-source 1 --name coingecko-price --script ../price.sh -
 
 	return cmd
 }
+
+// GetCmdCreateOracleScript implements the create oracle script command handler.
+func GetCmdCreateOracleScript(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-oracle-script (--name [name]) (--script [path_to_script]) (--owner [owner])",
```

Comment:
@taobun So are we going to use snake_case or camelCase for the things inside [...]?

Reply:
Let's go for dash-case. I think Cosmos-SDK usually uses that.